packages/lit-html/src/test/directives/guard_test.ts

```diff
@@ -163,4 +164,39 @@ suite('guard', () => {
 
     assert.equal(callCount, 2);
   });
+
+  test.only('guards directive from running', () => {
```

Comment:
Remove the `only`

Reply:
Made a PR to catch these: https://github.com/Polymer/lit-html/pull/1521

Reply:
Fixed.